cmd/kubeadm/app/phases/markcontrolplane/markcontrolplane.go

```diff
@@ -19,14 +19,19 @@ package markcontrolplane
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util/apiclient"
 )
 
 // MarkControlPlane taints the control-plane and sets the control-plane label
 func MarkControlPlane(client clientset.Interface, controlPlaneName string, taints []v1.Taint) error {
+	klog.V(1).Infof("[patchnode] Creating the Node API object %q if missing", controlPlaneName)
```

Comment:
[1] IMO we should add a comment explaining why we are explicitly creating a Node during init vs wait for the Kubelet to create it

Reply:
done. added a bigger comment on top of the function definition and added smaller comments near the calls for the readers to check the function definition.